README.md

```diff
@@ -34,11 +34,16 @@ a library use `npm install @dicy/core`. To install for command line access use
 Basic usage of the Command Line interface and library can be found below. For
 more detailed documentation please see the following pages.
 
--   [Configuration][configuration]
--   [Options][options]
--   [Commands][commands]
--   [Rules][rules]
--   [Events][events]
+-   [Comparison of Builders][] — A comparision of DiCy to [arara][] and
+    [latexmk][].
+-   [Commands][] — The commands that can be passed to DiCy.
```

Comment:
The terminology _subcommand_ seems more common (e.g., [google/subcommands](https://github.com/google/subcommands), [spf13/cobra](https://github.com/spf13/cobra), and [maxogden/subcommand](https://github.com/maxogden/subcommand)). So something like `[Commands][] — A list of DiCy's subcommands` is more simple.

Reply:
This is the only thing that I am probably not going to change right now even though I totally agree with you. Unfortunately I use the same terminology in the library API so I will need to do some code refactoring in order to change terminology. Therefore a separate PR will probably be needed.

Reply:
I understand the reason and agree that this should be a separate PR.

P.S. I really appreciate the built-in PythonTeX integration as I'll probably use PythonTeX in my thesis, which requires some `add_cus_dep` in latexmk. Thanks! :smile:

Reply:
Cool! Let me know how it goes with PythonTeX or anything else in regards to DiCy.